src/Compilers/CSharp/Test/Syntax/Parsing/ParserRegressionTests.cs

```diff
@@ -79,28 +77,13 @@ public void c024928()
             var tree = SyntaxFactory.ParseSyntaxTree(test);
         }
 
+#if !DEBUG 
```

Comment:
Ack! that's what I had and Aleksy recommended that I remove it in favor of this.  I can revert and fix the mistake that I had in that version.